### PR TITLE
fix: remove typography from dependencies

### PR DESCRIPTION
**Pull request by umbobabo**

### CODE REVIEW

**Comment by keithamus** [comment]
@umbobabo before I merge this, you should look at https://github.com/economist-components/component-link-button/pull/17 and merge it :smile:

**Comment by fabiosantoscode** [comment]
LGTM as a self-contained changeset

**Comment by umbobabo** [comment]
Linted css, added nesting to empower the internal styles against default values of bar-wrapper.
Can we merge?

File: src/index.css [hunk 1]
```diff
@@ -9,97 +8,96 @@
...
-       https://github.com/stylelint/stylelint/issues/857 (function-calc-no-unspaced-operator)
-    */
-    margin-top:
-      calc(
-        -1 *
+  & .subscribe-message__image {
+    display: none;
+  }
+
+  @media screen and (min-width: 950px) {
```
Comment (keithamus) [inline src/index.css]: You should nest the media queries in their respective selectors. It is much cleaner.

Comment (umbobabo) [inline src/index.css]: I think that wrapping in the media query is cleaner that vice-versa.
It make easier to understand what is happening on the page based on the state of browser resolution.
This comment is debatable and this part has been already approved as it was.

**Comment by angelorepole** [comment]
Let's raise Keith point somewhere and discuss with the team the ideal approach. For now this seems good to go.

**Pull request merged**